SPECS/golang/golang-1.15.spec

```diff
@@ -22,7 +22,6 @@ URL:            https://golang.org
 Source0:        https://dl.google.com/go/go%{version}.src.tar.gz
 Source1:        https://dl.google.com/go/go1.4-bootstrap-20171003.tar.gz
 Patch0:         go14_bootstrap_aarch64.patch
```

Comment:
Do you know, if this is still needed? The name suggests it might be relevant to an earlier version.

Reply:
This is still needed- this patch applies to the bootstrap version of Go, which is still 1.4 and still needs this patch.